Makefile

```
@@ -44,6 +44,35 @@ update: ## Update crowsnest (fetches and pulls repository changes)
 	@bash -c 'bin/build.sh --reclone'
 	${MAKE} build
 
+upgrade: ## Upgrade crowsnest from v4 to v5
```

checkmake (0.2.2)

[warning] Target body for "upgrade" exceeds allowed length of 5 (27).

(maxbodylength)